seas5/download_archive_seas5_tprate.py

```diff
@@ -0,0 +1,104 @@
+import os
+
+import pandas as pd
+import xarray as xr
+from ecmwfapi import ECMWFService
+
+from constants import CONTAINER_GLOBAL, SAS_TOKEN_DEV, STORAGE_ACCOUNT_DEV
+from utils.cloud_utils import upload_file
+
+server = ECMWFService("mars")
+
+
+def download_archive(year, bbox, td):
```

Comment:
Hmmm I'd vote for sticking with this function name since it does download the archival data for a given year. The looping just downloads archival data across multiple years. Since it's also within a more descriptive file name and directory that references `seas5`, `mars`, and `tprate`, I think adding any more specificity would be redundant.